docs/install/redhat.rst

```diff
@@ -28,11 +28,27 @@ from the CrateDB package repository::
     # Import the public GPG key for verifying the package signatures.
     sudo rpm --import https://cdn.crate.io/downloads/yum/RPM-GPG-KEY-crate
 
-    # Register with the CrateDB package repository.
-    sudo rpm -Uvh https://cdn.crate.io/downloads/yum/7/x86_64/crate-release-7.0-1.x86_64.rpm
-
-The command above will install the ``/etc/yum.repos.d/crate.repo`` package
-repository configuration file.
+To register with the CrateDB package repository, create a file called ``cratedb.repo``
+in the ``/etc/yum.repos.d/`` directory for RedHat based distributions, or in the
+``/etc/zypp/repos.d/`` directory for OpenSuSE based distributions, containing::
+
+    [cratedb-stable]
+    name=CrateDB RPM package repository - $basearch - Stable
+    baseurl=https://cdn.crate.io/downloads/yum/7/$basearch
+    enabled=1
```

Comment:
Do we want to enable it by default? ES doesn't for a reason, see the note at https://www.elastic.co/guide/en/elasticsearch/reference/current/rpm.html#rpm-repo.

Reply:
That's the rationale provided by ES:

> The configured repository is disabled by default. This eliminates the possibility of accidentally upgrading `elasticsearch` when upgrading the rest of the system. Each install or upgrade command must explicitly enable the repository as indicated in the sample commands above.

For us, it would be a change I guess, because it also was `enabled=1` before. I will be happy to follow ES's suggestions however, like you are proposing.

Reply:
I've also followed this advice, and amended the patch accordingly. Thanks.